test/system/200-pod.bats

```diff
@@ -478,26 +483,36 @@ spec:
         skip "only meaningful for rootful"
     fi
 
+    mkdir /dev/foo
+    mknod /dev/foo/null b 1 3
     local name1="resources1"
-    run_podman --cgroup-manager=systemd pod create --name=$name1 --cpus=5 --memory=10m
+    run_podman --cgroup-manager=systemd pod create --name=$name1 --cpus=5 --memory=5m --memory-swap=1g --cpu-shares=1000 --cpuset-cpus=0 --cpuset-mems=0 --device-read-bps=/dev/foo/null:1mb --device-write-bps=/dev/foo/null:1mb --blkio-weight-device=/dev/foo/null:123 --blkio-weight=50
     run_podman --cgroup-manager=systemd pod start $name1
     run_podman pod inspect --format '{{.CgroupPath}}' $name1
     local path1="$output"
     local actual1=$(< /sys/fs/cgroup/$path1/cpu.max)
     is "$actual1" "500000 100000" "resource limits set properly"
     local actual2=$(< /sys/fs/cgroup/$path1/memory.max)
-    is "$actual2" "10485760" "resource limits set properly"
+    is "$actual2" "5242880" "resource limits set properly"
```

Comment:
protip: error messages should always be unique. If/when this fails, the poor investigator will have no idea *what* resource failed. Even better, since this is growing so big, this is a good candidate for using `parse_table` and doing the check in a loop:
```bash
    run_podman pod inspect --format '{{.CgroupPath}}' $name1
    local cgroup_path="$output"

    expected_limits="
cpu.max         | 500000 100000
memory.max      | 5242880
memory.swap.max | 1068498944
io.max          | 1:3 rbps=1048576 wbps=1048576 riops=max wiops=max
"

    while read unit expect; do
        local actual=$(< /sys/fs/cgroup/$cgroup_path/$unit)
        is "$actual" "$expect" "resource limit: $unit"
    done < <(parse_table "$expected_limits")
    run_podman pod --cgroup-manager=systemd rm -f $name1
```